client/android-java/app/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.app"
```

Comment:
let's set a better app id - something like `com.stripe.android.payments`

Reply:
@bg-stripe thoughts?

I don't feel strongly either way, the project is just a container for the sample code imo.

Reply:
I don't feel strongly either, this is fine